src/test/scala/base/AsyncIntSpec.scala

```diff
@@ -12,11 +12,14 @@ import org.scalatest.matchers.should.Matchers
 import org.scalatest.time.Span
 import org.scalatest.wordspec.AsyncWordSpec
 
+import scala.concurrent.ExecutionContext
 import scala.concurrent.duration.*
 
 trait AsyncIntSpec[F[_]] extends AsyncWordSpec with AsyncIOSpec with Matchers with OptionValues with Eventually {
   override given patienceConfig: PatienceConfig = PatienceConfig(30.seconds, 500.millis)
 
+  override given executionContext: ExecutionContext = ExecutionContext.global
```

Comment:
this is only needed when using the `CatsResource` trait i thought?

see https://github.com/typelevel/cats-effect-testing/issues/300

Reply:
The last test in TopicLoaderIntSpec, `loadAndRun` sometimes fails and I'm trying to rule out it's to do with the resource being terminated before it can finish... but you're right, the global EC is only needed with the CatsResource.